src/Components/Artwork/Details.tsx

```diff
@@ -28,6 +30,7 @@ interface DetailsProps {
   hideArtistName?: boolean
   hidePartnerName?: boolean
   isHovered?: boolean
+  isMyCollectionArtwork?: boolean
```

Comment:
**suggestion (non-blocking):** What do you think about keeping this prop independent from the concept of My Collection? We could rename it to `showHighDemandIcon` and open the possibility to use the prop outside of My Collection in the future without confusion.

Reply:
Hmmmm. This is a great point. I will update the PR. Thank you 👍